app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -20,3 +20,28 @@
 .gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
   border-color: govuk-colour("white");
 }
+
+@include govuk-media-query($until: tablet) {
```

Comment:
This media query is repeated below. I'd suggest you wrap all of these changes inside one of them.

Reply:
will re-order